Src/WindowsServer/WindowsServer.Shared/Implementation/AzureHeartbeatProperties.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing;
+
+    internal class AzureHeartbeatProperties
+    {
+        internal static string HeartbeatPropertyPrefix = "azInst_"; // to ensure no collisions with base heartbeat properties
+
+        /// <summary>
+        /// Expected fields extracted from Azure IMS to add to the heartbeat properties. 
+        /// Set as internal for testing.
+        /// </summary>
+        internal readonly List<string> ExpectedAzureImsFields = new List<string>()
+        {
+            "osType",
```

Comment:
osType already added in base sdk ?

Reply:
Yes it is, and we can opt to remove this redundant data later on. However, I wanted to capture it going forward in case there was any use case for this osType value vs. the osType value returned by the .NET framework. (For instance, an Azure-specific osType moniker).